client/package.json

```diff
@@ -195,7 +194,8 @@
     "test-wdio-noCF": "env TEST_WDIO_SUITE=noCustomFields yarn run test-wdio",
     "build-storybook": "build-storybook",
     "storybook": "export STORYBOOK_ANET_DICTIONARY=/${ANET_DICTIONARY_NAME:-anet-dictionary.yml} ; start-storybook -s .. -p 6006",
-    "sim": "export ANET_DICTIONARY=../${ANET_DICTIONARY_NAME:-anet-dictionary.yml} SERVER_URL=${SERVER_URL:-'http://localhost:8080'} ; NODE_ENV=development webpack --mode development --config config/webpack.sim.dev.js && node build/anet.sim.dev.js"
+    "sim": "export ANET_DICTIONARY=../${ANET_DICTIONARY_NAME:-anet-dictionary.yml} SERVER_URL=${SERVER_URL:-'http://localhost:8080'} ; NODE_ENV=development webpack --config config/webpack.sim.dev.js && node build/anet.sim.dev.js",
+    "profile": "NODE_ENV=production webpack --config config/webpack.client.prod.js --profile --json > stats.json"
```

Comment:
The output file is in the order of 200MBs, we should probably output this file to a gitignored directory.